backend/api.py

```diff
@@ -1,12 +1,54 @@
 # create our little application :)
-from flask import Flask
+import configparser
+from flask import Flask, request
+import tweepy
 
 app = Flask(__name__)
 
+def are_you_the_keymaster():
+    config = configparser.ConfigParser()
+    config.read('keys.ini')
+    return config
 
-@app.route('/opinion')
+
+def twitter_handler(message):
+    twconf = are_you_the_keymaster()['twitter']
+    auth = tweepy.OAuthHandler(twconf['conkey'], twconf['consec'])
+    auth.set_access_token(twconf['atok'], twconf['atoksec'])
+    api = tweepy.API(auth)
+    return str(api.update_status(message))
+    #return str(api.verify_credentials())
+
+
+def choose_handler():
+    # TODO: add other handlers
+    def handler(message):
+        print("Sending message", str(message))
+        return twitter_handler
+    return handler
+
+
+def validate_message(message):
+    print("Testing message", message)
+    return message != ''
+
+
+def handle_message(message):
+    if not validate_message(message):
+        return 'Bad message!', 400
+
+    handler = choose_handler()
```

Comment:
Oh good call, but at least we have a handler message that can do that logic for us. That or we can always validate to make sure the message they send HAS to be valid for ALL of the handlers.